src/main/java/MatrixDiagonal.java

```diff
@@ -0,0 +1,17 @@
+public class MatrixDiagonal {
+    public static int[] getDiagonal(int[][] matrix) {
+        int[] diagonal = new int[matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            diagonal[i] = matrix[i][i];
+        }
+        return diagonal;
+    }
+
+    public static int[] getCounterDiagonal(int[][] matrix) {
+        int[] counterDiagonal = new int[matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            counterDiagonal[i] = matrix[i][matrix.length - 1 - i];
+        }
+        return counterDiagonal;
+    }
+}
```

Comment:
it was not the task here

Reply:
i can't understand this task... what did i do wrong?